scalafmt-tests-community/common/src/main/scala/org/scalafmt/community/common/TestHelpers.scala

```diff
@@ -58,9 +58,9 @@ object TestHelpers {
         val out1 = x1.formattedCode
         val lines2 = x1.formattedCode.count(_ == '\n')
         val (duration2, result2) = formatCode(out1)
-        def saveFormatted(): Unit = Files.writeString(
+        def saveFormatted(): Unit = Files.write(
```

Comment:
This method and the one below was added in JDK 11, but wasn't an issue for released artifacts since it's in the tests only

Reply:
should we instead add new flags to the `Compile /` so it doesn't affect Test?

Reply:
sounds good